scripts/dts/extract/globals.py

```diff
@@ -168,6 +172,9 @@ def find_node_by_path(nodes, path):
 
 
 def get_reduced(nodes, path):
+    if 'compat_count' not in get_reduced.__dict__:
```

Comment:
The name `compat_count` may be a bit misleading as the values stored here are the last assigned instance IDs, not numbers of occurrences of compats.

Reply:
Suggestions on name?

Reply:
Unfortunately as best I can tell there isn't a dts that hits this code path, so not sure what its trying to do anymore.

Reply:
`last_used_id` maybe?

Reply:
Done